package.json

```diff
@@ -41,7 +41,7 @@
     "predocumentation:dev": "node ./scripts/post-generator.js",
     "predocumentation:build": "node ./scripts/post-generator.js",
     "predocumentation:serve": "node ./scripts/post-generator.js",
-    "documentation:dev": "yarn build:es2015 --watch & next documentation-site",
+    "documentation:dev": "shx yarn build:es2015 --watch & next documentation-site",
```

Comment:
should the other scripts use `shx`? like the `documentation:build` below?

Reply:
yeah, that makes sense - added there too